decidim-core/app/views/decidim/links/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="wrapper external-link-page-content">
+  <h1 class="text-center"><%= t(".title") %></h1>
+  <div class="row columns large-centered large-8">
+    <div class="card">
+      <div>
+        <div class="card-section">
+          <p><%= t(".body_1") %></p>
+          <p><%= t(".body_2") %></p>
+        </div>
+        <div class="card-section">
+          <div class="row">
+            <div class="columns large-centered large-12">
+              <div class="heading4"><%= t(".external_link") %></div>
+              <div class="text-center external-link">
```

Comment:
In order to avoid adding additional styling, we could use the card styling for this box:

```ruby
<div class="card external-link">
  <div class="card__content text-center">
    <%= highlight_domain(params[:external_link]) %>
  </div>
</div>
```

After this change, we need to change the CSS for the background color (see other comment for the CSS).